mender_test.go

```diff
@@ -53,10 +53,12 @@ func Test_getArtifactName_noArtifactNameInFile_returnsEmptyName(t *testing.T) {
 
 	mender.artifactInfoFile = "artifact_info"
 
-	assert.Equal(t, "", mender.GetCurrentArtifactName())
+	artName, err := mender.GetCurrentArtifactName()
+	assert.NoError(t, err)
+	assert.Equal(t, "", artName)
 }
 
-func Test_getArtifactName_malformedArtifactNameLine_returnsEmptyName(t *testing.T) {
+func Test_getArtifactName_malformedArtifactNameLine_returnsError(t *testing.T) {
```

Comment:
Changed the name of the test function to reflect that it now checks for error instead of empty string. All tests are passing. Thinking this should not be an issue? 
Also, why are these test-functions of a different format. e.g. why the underscores?